tools/validators/instance_validator/validate/handler.py

```diff
@@ -197,29 +231,25 @@ def _TelemetryValidationCallback(
     validator: the telemetry validator that triggered the callback.
   """
 
-  print('Generating validation report ...')
-  current_time = datetime.datetime.now()
-  timestamp = current_time.strftime('%d-%b-%Y (%H:%M:%S)')
-  report = f'\nReport Generated at: {timestamp}\n'
-
+  print('[INFO]\tGenerating telemetry validation report.')
   if not validator.AllEntitiesValidated():
-    report += ('No telemetry message was received for the following '
```

Comment:
it does that still -- it is just refactored to the standard error message format, and put inside the FOR loop. instead of concatenating strings and printing them all at the end, i just print them as they arise. i think this is cleaner, since we dont use this type of string concat to do reporting anywhere else in the code base. wyt?

Reply:
I am not sure they are the same?
https://github.com/google/digitalbuildings/blob/master/tools/validators/instance_validator/validate/telemetry_validator.py#L79

Reply:
ping?

Reply:
this needs to stay, @trav3711 is updating the format, he can adjust as needed?

Reply:
Yeah I picked up the task to modify the report format. To avoid merge conflicts, telemetry validation report code should be left as is.